distributed_training/utils/misc.py

```diff
@@ -160,6 +161,10 @@ async def query_async():
 
         return await query_async()
 
+def generate_random_projection_matrix(seed, original_dim, projected_dim):
```

Comment:
Nice work @mikkelmedm. Do we still need all the projection functions and variables if we're not projecting anymore?

Reply:
Cool let's remove any projected functions or variables then just to keep the code clean